package.json

```diff
@@ -11,7 +11,7 @@
         "@nomiclabs/hardhat-ethers": "^2.0.1",
         "@nomiclabs/hardhat-etherscan": "^2.1.7",
         "@nomiclabs/hardhat-waffle": "^2.0.1",
-        "@openzeppelin/contracts": "^4.0.0",
```

Comment:
Relevant to item 1, pinned dependency versions to ensure we don't upgrade without realizing.